_site/index.html

```diff
@@ -346,6 +346,37 @@ <h3>T-Z</h3>
             </p>
           </div>
         </section>
+        <section class="block__word list">
+          <p class="word__signal word__signal__tool">
+            Tool of Oppression
+          </p>
+          <p class="word__title" id="ableism">
+            ableism
+            <span class="word__speech">
+              noun
+            </span>
+          </p>
+          <blockquote class="word__definition">
+            A system that places value on people’s bodies and minds based on societally constructed ideas of normalcy, intelligence and excellence. These constructed ideas of normalcy, intelligence and excellence are deeply rooted in anti-Blackness, eugenics and capitalism.
```

Comment:
>a system of oppression that: places value on people’s bodies and minds based on socially constructed ideas of normalcy, intelligence, and excellence—ideas founded from and deeply rooted in anti-Blackness, eugenics and capitalism; reinforces systemic power of non-disabled people over disabled people.

[I'm a lowercase wench who believes that we should start definitions in lowercase bc they're a continuation of the ableism, noun, a ... construction FIGHT ME]